helpers/filterSeats.js

```diff
@@ -23,6 +26,12 @@ module.exports = function(dust) {
     var roomRates = params.roomRates;
     var bandColours = params.bandColours;
     var sortByInput = params.sortBy;
+    var seatLegend = params.seatLegend.seatLegend.en;
```

Comment:
having a key that is `seatLegend.seatLegend` seems redundant. Can we fix this further up at all (from the API structure maybe)?